src/riak_object.erl

```
@@ -129,6 +130,19 @@ new(B, K, V, MD) when is_binary(B), is_binary(K) ->
 newts(B, K, V, MD) ->
     new_int2(B, K, V, MD).
 
+-spec is_ts(riak_object()) -> {'true', pos_integer()} | 'false'.
+is_ts(RObj) ->
+    check_for_ddl(get_contents(RObj)).
+
+check_for_ddl([{Metadata, _V}]) ->
```

Write Once objects never have siblings. Maybe better to crash than to silently ignore that, and lose data presumably if it ever happens.